server/TracyView_FindZone.cpp

```diff
@@ -1362,6 +1362,23 @@ void View::DrawFindZone()
         SmallCheckbox( "Show zone time in frames", &m_findZone.showZoneInFrames );
         ImGui::Separator();
 
+        ImGui::AlignTextToFramePadding();
+        TextDisabledUnformatted( "Filter user text:" );
+        ImGui::SameLine();
+        bool filterChanged = m_userTextFilter.Draw( ICON_FA_FILTER "###resultFilter", 200 );
+
+        ImGui::SameLine();
+        if( ImGui::Button( ICON_FA_DELETE_LEFT " Clear" ) )
+        {
+            m_userTextFilter.Clear();
+            filterChanged = true;
+        }
+        ImGui::Separator();
+        if( filterChanged )
+        {
+            m_findZone.ResetGroups();
```

Comment:
I'm not sure if this is the correct way to "dirty" the zone list to cause it to redraw but this seems to work at least.